HelpSource/Reference/ServerPluginAPI.schelp

```diff
@@ -0,0 +1,189 @@
+title:: Server Plugin API
```

Comment:
Okay, here's the real reason I've been resisting automated documentation: I have a bit of a grudge against it because I really dislike the way those docs tend to come out. They're usually organized way too much like code and not enough like a useful manual that prioritizes features of an API based on how useful and necessary they are to readers. If we develop such a system, I request that we ensure that its output is actually human-usable.

> neither unroll macros, nor the inline functions are actually part of the actual plugin interface ... the rest is basically helper functionality

But this helper functionality is where the actually useful info is for writing plugins! I want to write documentation that shows people the necessary info on how to do things. In my personal experience, I was seeing all sorts of functions and macros like ZXP and sc_clip and sc_mod and wondering where the hell they came from, and I'm writing this up to save future developers from that confusion. If my use of the term "API" isn't technically accurate, then okay, I can change it.